MailManager/src/main/java/inc/softserve/health/MailManagerHealthCheck.java

```diff
@@ -35,18 +35,20 @@ public MailManagerHealthCheck(@ESPConnectorAdmin String ESPConnectorAdminUrl,
 
     protected Result check() {
```

Comment:
This method is now awesome. Are you up to do it using `CompletableFuture`? :smiley: